src/TaskRunner/Commands/DrupalCommands.php

```diff
@@ -145,7 +145,7 @@ protected function getToolkitSettingsBlock() {
   'driver' => 'mysql',
 );
 {$additionalSettings}
-\$settings['hash_salt'] = '{$hashSalt}';
+\$settings['hash_salt'] = getenv('DRUPAL_HASH_SALT') ? getenv('DRUPAL_HASH_SALT') : '{$hashSalt}';
```

Comment:
We could check on strict type here:
```
 \$settings['hash_salt'] = getenv('DRUPAL_HASH_SALT') !== FALSE ? getenv('DRUPAL_HASH_SALT') : '{$hashSalt}';
```

Reply:
Okey, I'll check/test on Monday. Also need to fix the PHPunit tests